rakefile.rb

```diff
@@ -162,6 +176,12 @@
   run_msbuild solution, "Build", msbuild_command
 end
 
+desc "GitLink PDB's"
+exec :gitlink => [:build] do |cmd|
+  cmd.command = gitlink_command
+  cmd.parameters ". -u #{repo_url} -ignore " + gitlink_ignores.join(",")

```

Comment:
The URL turned out to be the only thing that GitLink coudn't work out for itself, perhaps because I'm using SSH instead of HTTP.